src/test/resources/com/cognifide/gradle/aem/test/debug/additional/buildSrc/build.gradle.kts

```diff
@@ -5,5 +5,5 @@ repositories {
 }
 
 dependencies {
-    implementation("com.cognifide.gradle:aem-plugin:8.1.2")
+    implementation("com.cognifide.gradle:aem-plugin:9.0.0")
```

Comment:
I think you could drop the `buildSrc` folder of all the integration tests if you add a `.withPluginClasspath()` call to the `GradleRunner` that you configure for the integration tests. This would also get rid of `test` needing to depend on `publishToMavenLocal` as this should automatically make all your gradle plugins available for the build of the `GradleRunner`. At least it worked on my machine…

Reply:
yep I know, just waits in the task queue ;)

I have these tests since that method was not even available ;)